### PR TITLE
bird: add page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.0.10

### CODE REVIEW

File: pages/common/bird.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# bird
+
+> BIRD Internet Routing Daemon.
+> Routing daemon with support for BGP, OSPF, Babel and others.
+> More information: <https://bird.network.cz/>.
+
+- Start Bird with a specific configuration file:
+
+`bird -c {{bird.conf}}`
```
Comment (CleanMachine1) [inline pages/common/bird.md:9]:
```suggestion
`bird -c {{path/to/bird.conf}}`
```

File: pages/common/bird.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+- Start Bird as a specific user and group:
+
+`bird -c {{bird}} -g {{bird}}`
```
Comment (CleanMachine1) [inline pages/common/bird.md:13]: I think command is wrong
```suggestion
`bird -u {{username}} -g {{group}}`
```

File: pages/common/bird.md (hunk 1 above)
Comment (marek22k) [reply pages/common/bird.md]: Thanks for the hint!

File: pages/common/bird.md (hunk 2 above)
Comment (marek22k) [reply pages/common/bird.md]: How I came up with that is beyond me - thanks for the correction.

File: pages/common/bird.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+> Routing daemon with support for BGP, OSPF, Babel and others.
+> More information: <https://bird.network.cz/>.
+
+- Start Bird with a specific configuration file:
+
+`bird -c {{path/to/bird.conf}}`
+
+- Start Bird as a specific user and group:
+
+`bird -u {{bird}} -g {{bird}}`
```
Comment (CleanMachine1) [inline pages/common/bird.md:13]:
```suggestion
`bird -u {{username}} -g {{group}}`
```
quite unclear placeholders just using bird, or is bird:bird just the most commonly used user:group combination?

Comment (marek22k) [reply pages/common/bird.md]: These are the default user and group generated when installing on Debian, but yes here it is unclear.

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**